scripts/grab-mergecommits/main.go

```diff
@@ -4,27 +4,21 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
-	"regexp"
 	"strings"
 	"time"
 
 	"github.com/ActiveState/cli/internal/errs"
 	"github.com/ActiveState/cli/internal/exeutils"
-	"github.com/ActiveState/cli/internal/rtutils/p"
 	"github.com/ActiveState/cli/internal/sliceutils"
-	"github.com/andygrunwald/go-jira"
-	"github.com/google/go-github/v45/github"
+	github_helpers "github.com/ActiveState/cli/scripts/internal/github-helpers"
+	jira_helpers "github.com/ActiveState/cli/scripts/internal/jira-helpers"
```

Comment:
This is snake case, but regardless Go doesn't like either for package names. I try to avoid it but there's times where doing things the idiomatic way just doesn't make sense imo. I guess you could call this `ghhelpers` or `githubhelpers` but how is that an improvement?

Reply:
Yeah fair enough, not a big deal anyways